part/part_test.go

```diff
@@ -821,3 +817,73 @@ func benchmark_txn_delete_batch(b *testing.B, batchSize int) {
 	}
 	b.ReportMetric(float64(b.N)/b.Elapsed().Seconds(), "objects/sec")
 }
+
+func Benchmark_Get(b *testing.B) {
+	const numObjectsToInsert = 1000
```

Comment:
this shadows the global const, intentionally?

Reply:
Accidental. Fixed.